src/subdomains/staking/application/dto/output/balance.output.dto.ts

```diff
@@ -1,10 +1,15 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { StakingStatus } from 'src/subdomains/staking/domain/enums';
 
 export class DepositAddressBalanceOutputDto {
   @ApiProperty()
-  depositAddress: string;
+  address: string;
```

Comment:
I think it's kind of confusing to return either the deposit or the user address. I would fix it or return no address. WDYT?